contracts/FIND.cdc

```diff
@@ -107,7 +108,9 @@ pub contract FIND {
 		let trimmedInput = FIND.trimFindSuffix(input)
 
 		if FIND.validateFindName(trimmedInput) {
-			return FIND.lookupAddress(trimmedInput)
+			if let network = self.account.borrow<&Network>(from: FIND.NetworkStoragePath) {
+				return network.lookup(trimmedInput)?.owner?.address
+			}
 		}
```

Comment:
```suggestion
return nil
		}
```
we might have to return nil here otherwise it will fall into resolving as address and panic